### PR TITLE
Setup script

### PR DESCRIPTION
**Pull request by adeelq**

### CODE REVIEW

File: code/Dotdigitalgroup/Email/sql/email_connector_setup/mysql4-install-6.3.0.php [hunk 1]
```diff
@@ -0,0 +1,1522 @@
...
+            'store_id'
+        )
+    )
+    ->where('customer_id =?', 0)
+    ->where('subscriber_status =?', 1);
+$insertArray = array('email', 'is_subscriber', 'subscriber_status', 'store_id');
+$sqlQuery = $select->insertFromSelect($contactTable, $insertArray, false);
+$installer->getConnection()->query($sqlQuery);
+
+//Update contacts with customers that are subscribers
```
Comment (adeelq) [inline code/Dotdigitalgroup/Email/sql/email_connector_setup/mysql4-install-6.3.0.php]: @dotZygis 433 to 449 is the change

**Comment by cdiacon** [comment]
@adeelq can you replace the crossupdate for orders like we did for subscribers ?

https://github.com/dotmailer/dotmailer-magento-extension/blob/6df0aab6afb170f37da0729aa136263434fabc1d/code/Dotdigitalgroup/Email/sql/email_connector_setup/mysql4-install-6.0.0.php#L542

should be no crossupdate in install scripts.

**Comment by cdiacon** [comment]
typo change message to "Store Name"

https://github.com/dotmailer/dotmailer-magento-extension/blob/6df0aab6afb170f37da0729aa136263434fabc1d/code/Dotdigitalgroup/Email/sql/email_connector_setup/mysql4-install-6.0.0.php#L1447

**Comment by cdiacon** [comment]
remove index for the message column - don't think this should be indexed 

https://github.com/dotmailer/dotmailer-magento-extension/blob/6df0aab6afb170f37da0729aa136263434fabc1d/code/Dotdigitalgroup/Email/sql/email_connector_setup/mysql4-install-6.0.0.php#L1507

**Comment by cdiacon** [comment]
typo error

https://github.com/dotmailer/dotmailer-magento-extension/blob/6df0aab6afb170f37da0729aa136263434fabc1d/code/Dotdigitalgroup/Email/sql/email_connector_setup/mysql4-install-6.0.0.php#L307

**Comment by cdiacon** [comment]
add index to "contact_id" 


https://github.com/dotmailer/dotmailer-magento-extension/blob/6df0aab6afb170f37da0729aa136263434fabc1d/code/Dotdigitalgroup/Email/sql/email_connector_setup/mysql4-install-6.0.0.php#L33

**Comment by cdiacon** [comment]
email_importer table : 
* add index "import id" 
* add index to "import_mode" 
* remove index "message"

https://github.com/dotmailer/dotmailer-magento-extension/blob/6df0aab6afb170f37da0729aa136263434fabc1d/code/Dotdigitalgroup/Email/sql/email_connector_setup/mysql4-install-6.0.0.php#L1321

**Comment by cdiacon** [comment]
email_wishlist table : 
* add index to "store_id"

https://github.com/dotmailer/dotmailer-magento-extension/blob/6df0aab6afb170f37da0729aa136263434fabc1d/code/Dotdigitalgroup/Email/sql/email_connector_setup/mysql4-install-6.0.0.php#L873

**Comment by cdiacon** [comment]
install script 6.3.0 should include also the changes from this file 

https://github.com/dotmailer/dotmailer-magento-extension/blob/9584861175d53d3c942e4452226b252d711943be/code/Dotdigitalgroup/Email/sql/email_connector_setup/mysql4-upgrade-6.1.0-6.1.1.php#L17-L17

**Pull request merged**